README.md

```diff
@@ -1,2 +1,150 @@
 # VR-Flight-Simulator
 Srishti 2019
+## IMAGES OF THE PROJECT:
+
+### while paying
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/while%20playing.jpeg)
+
+### plane mesh made from wire
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/plane.jpeg)
+
+### Project Mechanical design
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Project%20Mechanical%20design.jpeg)
+
+### Game made on unity
+![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Game.jpeg)
+***
+## ABSTRACT:
+
+The project we made is known as vr flight simulator.It is a simulation of real life plane experience.
+In this project We had used a ultrasonic sensor, a bump sensor, a potentiometer sensor, a gyroscope
+sensor (esp8266), and a esp9250 module to make a joystick and a throttle handle. The joytick physically
+moved in vertical circular motion in left write direction, and it moved front and backwards producing 
+respective motion in game. The throttles went front and backwards to accelerate and decelerate the plane
+respectively.A game on pc was developed by us which contains a plane and some objects which can be shot.
+***
+## MOTIVATION:
+
+Our motivation for doing this project was the creativity of the idea and our fascination with the end result.
+***
+## MECHANICAL ASPECT OF DESIGN:
+
+Our mechanical parts consists of
+
+1. joystick
+2. throttle
+
+Explaination :-
```

Comment:
The spelling of Explanation.

Remove the upper line write the name and description simultaneously.